### DISCUSSION TITLE
How long does it take to retrieve a 32g file？

### DISCUSSION THREAD

**Discussion by PerrywLiu** [filecoin-project/lotus#7805] [General Q&A] [answered]
How long does it take to retrieve a 32g file？

**Comment by PerrywLiu** [comment]
How long does it take to retrieve and unseal a 32g file？

**Comment by RobQuistNL** [comment, marked as answer]
Retrieve: Depends on your the internet speed of both machines

Unseal: Depends on the hardware of the host. Usually you can expect it to be about 4 hours.

**Comment by Reiers** [comment]
Hi @PerrywLiu 

Thanks for the report. The is not an issue.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing ticket.

**Comment by LexLuthr** [comment]
@PerrywLiu The unseal operation should take about the same time as the sealing operation. You can try to benchmark it in calibnet for your hardware. The unseal steps follow almost the same steps as sealing. The default behaviour in Lotus is to keep a copy of the original data. So, you might need to unseal a sector to provide back the data to the client.

**Reply by LexLuthr** [reply]
I am archiving this thread as there has been no new input for more than 7 days.
